coalib/bearlib/aspects/Spelling.py

```diff
@@ -1,72 +1,65 @@
-from coalib.bearlib.aspects import Root
+from coalib.bearlib.aspects import Root, Taste
 
 
 @Root.subaspect
 class Spelling:
     """
-    This aspect describes spelling of your source code.
+    This aspect checks for misspelled words in your docs.
     """
     class docs:
         example = """
-        `Coala`, `AspectsYEAH`
+        'Tihs si surly som incoreclt speling.
+        `Coala` is always written with a lowercase `c`.
         """
-        example_language = 'All'
+        example_language = 'English'
         importance_reason = """
-        They are words we always want to be written the way we want like the
-        name of our app eg: `coala`...
+        Words should always be written as the are supposed to be,
+        standartization facilitates communication.
```

Comment:
`standartization` ? shoudln't it be `standardization`? sorry if i don't know the word